src/data/code-samples/en/accessibility-announcement/jetpack-compose.md

```diff
@@ -0,0 +1,24 @@
+# Accessibility announcement - Jetpack Compose
+
+In Jetpack Compose, to notify `Composable` state changes, you can use the [`liveRegion`](https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/package-summary.html#(androidx.compose.ui.semantics.SemanticsPropertyReceiver).liveRegion()) property from [`semantics`](https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/package-summary#(androidx.compose.ui.Modifier).semantics(kotlin.Boolean,kotlin.Function1)) block modifier.
+
+You can choose from two options for `liveRegion`:
+
++ [`LiveRegionMode.Polite`](https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/LiveRegionMode#Polite()) , which waits for the speech announcement in progress to complete
```

Comment:
Can you use `1.` and `2.` instead of `+`

Reply:
Resolved, changed list to numbered.